client/src/actions.js

```diff
@@ -0,0 +1,4 @@
+import * as types from './constants/ActionTypes'
+
+export const userLogin = id => ({ type: types.USER_LOGIN, id })
+export const userLogout = () => ({ type: types.USER_LOGOUT })
```

Comment:
good idea. I'll be doing some login refactoring soon if you don't mind me doing it then.